esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -1146,7 +1157,9 @@ bool CWsWorkunitsEx::onWUWaitCompiled(IEspContext &context, IEspWUWaitRequest &r
     {
         StringBuffer wuid = req.getWuid();
         WsWuHelpers::checkAndTrimWorkunit("WUWaitCompiled", wuid);
+        ensureWsWorkunitAccess(context, wuid.str(), SecAccess_Full);
```

Comment:
I am not sure about SecAccess_Full in this and other places, should we be less restrictive?

Reply:
This does not change a WU.